partition/roles/dhcp/templates/dhcpd.conf.j2

```diff
@@ -15,13 +15,13 @@ subnet {{ subnet.network }} netmask {{ subnet.netmask }} {
   range {{ subnet.range.begin }} {{ subnet.range.end }};
 {% endif %}
 {% for option in subnet.options | default([]) %}
-  option {{ option }};
```

Comment:
The interface should not be changed again, this would be breaking for mini-lab. Just omit the `option` string when defining the options.

Reply:
Also option is actually required as otherwise a user can mess up the config with arbitrary values easily.

Reply:
I reverted these changes.